skins/oasis/js/BackgroundChanger.js

```diff
@@ -72,7 +71,26 @@ define('wikia.backgroundchanger', function()  {
 			} else {
 				$('body').removeClass('background-dynamic background-not-tiled background-fixed');
 			}
-		});
+		};
+
+		if (!$.isArray(doc.loadedCss)) {
+			doc.loadedCss = [];
```

Comment:
Is this if needed? Do we need to put this variable in `document` object? Can't it be declared in `wikia.backgroundchanger` module as `var loadedCss = []`?